packages/plugins/geth/src/devtxs.ts

```diff
@@ -4,18 +4,24 @@ import Web3 from "web3";
 import constants from "embark-core/constants.json";
 export default class DevTxs {
   private embark: Embark;
+  private blockchainConfig: any;
   private events: Events;
   private logger: Logger;
   private web3?: Web3;
   private regularTxsInt?: NodeJS.Timeout;
   constructor(embark: Embark) {
     this.embark = embark;
+    this.blockchainConfig = this.embark.config.blockchainConfig;
     this.logger = embark.logger;
     this.events = embark.events;
   }
   public async init() {
-    const provider = await this.events.request2("blockchain:client:provider", "ethereum");
-    this.web3 = new Web3(provider);
+    const { rpcHost, rpcPort, wsRPC, wsHost, wsPort } = this.blockchainConfig;
```

Comment:
Defaults have been fully merged at this point?

Reply:
good catch, I wanted to ensure this merged defaults here. will fix